packages/nextjs-demo/src/app/asr/page.tsx

```diff
@@ -97,35 +129,62 @@ const AsrComponent: React.FC<AsrComponentProps> = ({ name, link, id, costPerMinu
   return (
     <div className="ml-2">
       <p className="text-xl font-bold mt-2">
-        <a href={link}>{name}</a>
+        <a className="underline" href={link}>
+          {name}
+        </a>
       </p>
       <div className="text-sm">
         <span className="font-bold">Cost: </span>
-        <a href={`${link}/pricing`}>${costPerMinute}/min</a>
+        <a className={`underline ${computeCostColor(costPerMinute)}`} href={`${link}/pricing`}>
+          ${costPerMinute}/min
+        </a>
+      </div>
+      <div className="text-sm">
+        <span className="font-bold">Partial Latency: </span>
+        {computeLatency(partialLatency).toFixed(0)} ms
       </div>
       <div className="text-sm">
-        <span className="font-bold">Latency: </span>
-        {latency.toFixed(0)} ms
+        <span className="font-bold">Final Latency: </span>
+        {computeLatency(finalLatency).toFixed(0)} ms
       </div>
       <div className="text-sm">
         <span className="font-bold">WER: </span>
-        {wer.toFixed(3)}
+        {computeWer(output, transcript).toFixed(3)}
       </div>
       <textarea cols={80} rows={5} ref={textarea}></textarea>
     </div>
   );
 };
 
+const ButtonComponent: React.FC<{ onClick: () => void; disabled: boolean; children: React.ReactNode }> = ({
+  onClick,
+  disabled,
+  children,
+}) => (
+  <button
+    onClick={onClick}
+    disabled={disabled}
+    className={`ml-2 rounded-md ${
+      disabled ? 'bg-gray-300' : 'bg-fixie-fresh-salmon hover:bg-fixie-ripe-salmon'
+    } px-3 py-2 text-sm font-semibold text-white shadow-sm focus-visible:outline focus-visible:outline-2 focus-visible:outline-offset-2 focus-visible:outline-fixie-fresh-salmon`}
```

Comment:
You can also use https://www.npmjs.com/package/classnames for this:

```ts
className={classNames('ml-2 rounded-md px-3 py-2 text-sm font-semibold text-white shadow-sm focus-visible:outline focus-visible:outline-2 focus-visible:outline-offset-2 focus-visible:outline-fixie-fresh-salmon', disabled ? 'bg-gray-300' : 'bg-fixie-fresh-salmon hover:bg-fixie-ripe-salmon')};
```

Not hugely different but some people prefer it to the string interpolation.